src/App.tsx

```diff
@@ -15,71 +26,98 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+    selectedClass: 'has-background-success-light',
+    noOneSelected: 'No goods selected',
+    addButtonText: '+',
+    removeButtonText: '-',
```

Comment:
redundant a lot of states.
enough to keep in state only selected good
classes, text content render conditionally